app/javascript/react/screens/App/Overview/components/InfrastructureMappingsList/InfrastructureMappingsList.js

```diff
@@ -45,20 +45,53 @@ class InfrastructureMappingsList extends React.Component {
       transformationMappingsMutable: this.state.transformationMappingsMutable
     });
   };
+
   closeExpand = mapping => {
     mapping.expanded = false;
     this.setState({
       transformationMappingsMutable: this.state.transformationMappingsMutable
     });
   };
 
+  renderEmptyState = ({ divider }) => (
+    <React.Fragment>
+      {divider && <hr style={{ borderTopColor: '#d1d1d1' }} />}
```

Comment:
rather than specifying the color inline and by hash, consider creating a class and using the closest `$color-pf-foo` here's an example of blueeee https://github.com/manageiq/miq_v2v_ui_plugin/blob/bc5a6630477159b4fcfdcbc9b9a3799613882315/app/javascript/react/screens/App/Overview/components/InfrastructureMappingsList/InfrastructureMappingsList.scss#L107